text/0000-compiler-requirements.md

```diff
@@ -2,18 +2,45 @@
 
 ## Current State
 
+The entire font compilation pipeline is written in Python, with some parts being written in Cython for speed.
+
 ### Current Tools
 
+fontmake. It's the glue that binds together ufo2ft, fontTools and some other codebases to take any of the usual font formats (UFOs, Designspace + UFOs and Glyphs.app files) and produce font binaries (TTF, CFF(2); static, interpolated static, variable).
```

Comment:
nanoemoji, though limited in scope, may bear mention. Significant primarily because it uses python to write ninja and that has proven to be very effective for those parts of the process that are readily amenable to ninja.

A section on low hanging fruit, such as introducing parallelism to fontmake, would be interesting.

An interesting decision is coming: should we try to speed up fontmake before, or instead of, waiting on a new compiler.